src/main/kotlin/no/nav/syfo/client/azuread/AzureAdClientMetric.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.syfo.client.azuread
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Counter.builder
+import no.nav.syfo.metric.METRICS_NS
+import no.nav.syfo.metric.METRICS_REGISTRY
+
+const val CALL_AZUREAD_BASE = "${METRICS_NS}_call_azuread"
+
+const val CALL_AZUREAD_TOKEN_SYSTEM_BASE = "${CALL_AZUREAD_BASE}_token_system"
+const val CALL_AZUREAD_TOKEN_SYSTEM_CACHE_HIT = "${CALL_AZUREAD_TOKEN_SYSTEM_BASE}_cache_hit_count"
+const val CALL_AZUREAD_TOKEN_SYSTEM_CACHE_MISS = "${CALL_AZUREAD_TOKEN_SYSTEM_BASE}_cache_miss_count"
+
+val COUNT_CALL_AZUREAD_TOKEN_SYSTEM_CACHE_HIT: Counter = builder(CALL_AZUREAD_TOKEN_SYSTEM_CACHE_HIT)
```

Comment:
Trenger vi ikke metrikker for token-kall som feiler/er OK? Vi har kanskje ikke det andre steder heller? 🤔

Reply:
Det var ikke her i denne klienten, men jeg kan godt legge det til etterpå.